src/components/Global/BaseDialog.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-dialog v-model="show" persistent max-width="650" :scrollable="scrollable" elevation scrim="surface-1">
+    <v-card :id="id" class="dialog-card" color="surface-1">
+      <v-card-title class="d-flex justify-space-between pt-4">
+        <h1 v-if="title">{{ title }}</h1>
+        <slot name="title" />
```

Comment:
This should probably be a v-if/else, so that you can use the prop for simple use-cases, or the slot for more complex use cases. If you're going to use both then they should be named different/uniquely.